src/components/Dashboard/TodoList.vue

```diff
@@ -1928,32 +1922,12 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
         this.$router.push({ name: Routes.CONTINUATION_OUT, params: { filingId: item.filingId.toString() } })
         return
 
-      case FilingTypes.CORRECTION:
-        // see also ItemHeaderActions.vue:correctThisFiling()
-        switch (item.correctedFilingType) {
-          case FilingNames.ALTERATION:
-          case FilingNames.CHANGE_OF_REGISTRATION:
-          case FilingNames.CORRECTION:
-          case FilingNames.INCORPORATION_APPLICATION:
-          case FilingNames.REGISTRATION:
-          case FilingNames.SPECIAL_RESOLUTION:
-            // resume correction via Edit UI
-            navigateToCorrectionEditUi(this.getEditUrl, this.getIdentifier)
-            return
-
-          case FilingNames.CHANGE_OF_ADDRESS:
-          case FilingNames.CHANGE_OF_DIRECTORS:
-            if (this.isBaseCompany || this.isCoop) {
-              // resume correction via Edit UI if current type is BEN/BC/CC/ULC/C/CBEN/CCC/CUL or CP
-              navigateToCorrectionEditUi(this.getEditUrl, this.getIdentifier)
-              return
-            }
-            break
-        }
-        // resuming correction is not supported for all other filings
-        // eslint-disable-next-line no-console
-        console.log('doResumeFiling(), invalid correction type for item =', item)
+      case FilingTypes.CORRECTION: {
+        // resume correction via Edit UI
+        const correctionUrl = `${this.getEditUrl}${this.getIdentifier}/correction/?correction-id=${item.filingId}`
+        navigate(correctionUrl)
```

Comment:
As above, if we get this far then we can assume that resuming the correction for this filing type is allowed, so we just do it. No need for more checks.